app/views/shared/_navbar.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="navbar">
   <div class="d-flex align-items-center m-0">
-    <i id="logo" class="fas fa-wrench"></i>
-    <a href="/" class="my-1 ml-1" id="main-name"> Solve It</a>
+   <a href="/" id="logo" class="fas fa-wrench"></a>
+    <%# <a href="/" class="my-1 ml-1" id="main-name"> Solve It</a> %>
```

Comment:
Esta parte parece comentada. Estamos ocultando el nombre Solve it, esta bien eso? 
https://share.getcloudapp.com/2Nuvdrk0